src/layer/S3SLSTRLayer.ts

```diff
@@ -12,29 +13,58 @@ type S3SLSTRFindTilesDatasetParameters = {
 
 export class S3SLSTRLayer extends AbstractSentinelHubV3Layer {
   public readonly dataset = DATASET_S3SLSTR;
+  protected maxCloudCoverPercent: number;
+  protected orbitDirection: OrbitDirection | null;
+  protected view: 'NADIR' | 'OBLIQUE';
+
+  public constructor(
+    instanceId: string | null,
+    layerId: string | null = null,
+    evalscript: string | null = null,
+    evalscriptUrl: string | null = null,
+    dataProduct: string | null = null,
+    title: string | null = null,
+    description: string | null = null,
+    maxCloudCoverPercent: number | null = 100,
+    orbitDirection: OrbitDirection | null = null,
+    view: 'NADIR' | 'OBLIQUE' = 'NADIR',
+  ) {
+    super(instanceId, layerId, evalscript, evalscriptUrl, dataProduct, title, description);
+    this.maxCloudCoverPercent = maxCloudCoverPercent;
+    this.orbitDirection = orbitDirection;
+    this.view = view;
+  }
+
+  protected async updateProcessingGetMapPayload(payload: ProcessingPayload): Promise<ProcessingPayload> {
+    payload.input.data[0].dataFilter.maxCloudCoverage = this.maxCloudCoverPercent;
+    return payload;
+  }
+
+  protected getWmsGetMapUrlAdditionalParameters(): Record<string, any> {
+    return {
+      maxcc: this.maxCloudCoverPercent,
```

Comment:
Would `orbitDirection` and `view` also be needed here?

Reply:
Maybe, not sure to be honest... I can't find them as WMS parameters so I'm no sure if they are supported. 

As discussed, we should probably leave this for now, and will revisit this question when we make the test suite for S-3.

Reply:
Agreed. Resolving.